pkg/tinydtls/Makefile.include

```diff
@@ -3,7 +3,7 @@ PKG_BUILDDIR ?= $(PKGDIRBASE)/tinydtls
 INCLUDES += -I$(PKG_BUILDDIR)
 
 ifeq ($(TOOLCHAIN), llvm)
-  CFLAGS += -Wno-gnu-zero-variadic-macro-arguments -Wno-unused-function
+  CFLAGS += -Wno-gnu-zero-variadic-macro-arguments -Wno-unused-function -Wno-unused-parameter
```

Comment:
I'm sorry, if you touch this line, it'll also have to go into Makefile.

Reply:
somehow tinydtls only fails on native, and setting this in `Makefile` doesn't help? @kaspar030 any idea why, for libcoap it was okay to simply set it in `Makefile`.

Reply:
btw. it only fails if `DEVELHELP` is **not** set for native, if it is enabled the example builds.

Reply:
ah DEBUG output, I guess

Reply:
with DEVELHELP the parameter is used for some debug message and without its unused, arrrggg